org.eclipse.jdt.ls.debug/src/org/eclipse/jdt/ls/debug/adapter/DebugRequestHandleStage.java

```diff
@@ -0,0 +1,17 @@
+/*******************************************************************************
+ * Copyright (c) 2017 Microsoft Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Microsoft Corporation - initial API and implementation
+ *******************************************************************************/
+
+
+package org.eclipse.jdt.ls.debug.adapter;
+
+public enum DebugRequestHandleStage {
```

Comment:
Cannot see the need of modeling the procedure into stages. We don't need pipeline yet.